cas-server-webapp/src/main/webapp/WEB-INF/view/jsp/default/ui/casLoginView.jsp

```diff
@@ -109,6 +109,19 @@
     <div class="sidebar-content">
         <p><spring:message code="screen.welcome.security" /></p>
 
+        <c:if test="${!empty pac4jUrls}">
+            <div id="list-providers">
+                <h3><spring:message code="screen.welcome.label.loginwith" /></h3>
+                <form>
+                    <ul>
```

Comment:
Are these urls that would be decorated with logo images of various providers? don't see the images in PR :)